### DISCUSSION TITLE
Domain Name Change - Data Migration

### DISCUSSION THREAD

**Discussion by seannaswell** [zotonic/zotonic#3643] [General]
So, I have started populating a site with data, which at this point is over 200 items. It consists of Categories and Predicates, Media items and Location data, lots of information has been entered.

However, the domain name of the site needs to change. Other than the schema and zotonic.site.config file, does anything else need to change? 

Any hints for how to change the domain used for a site without having to repopulate data would be much appreciated...

**Comment by geoidesic** [comment]
Did you manage to find this out?

**Comment by mworrell** [comment]
Nothing else need to change.

Unless you copied verbatim URLs in text content or shared links to the old domain name.

We often keep the older sitename in the `hostalias` config.
